### ISSUE TITLE
Any plans to release the next version?

### ISSUE THREAD

**Issue by kadishmal** [usablica/intro.js#453]
The last 1.0.0 was released 10 months ago. There are 9 changes merged with some fixes. Any plan to release them?

Another question: there seem to so many issues and unbelievable amount of PRs. Is this library maintained at all? Or is the author trying not to involve anybody but himself?

Intro.js is awesome, but it has so many issues especially with optioning the tooltips, interactions with the target element. I saw several related issues and PRs that resolve some of them, why aren't they merged?

Or should we switch to something else and label with repo as **NOT MAINTAINED**?

**Comment by kleinph** [comment]
This is great news!
I was also worried that this repo is defacto unmaintained, but a few weeks ago there was some activity again. A few PRs were merged, but nothing happened since then.

If you fork this repo can you also merge  #448 and #441?

Thanks

**Comment by afshinm** [comment]
Hey guys,
I'm planning to release a new version asap. I was busy with some projects in last months so I couldn't maintain the project but I'm going to merge PRs again.

Stay tuned :)

**Comment by afshinm** [comment]
@kadishmal Can you please help me merge PRs?

**Comment by kadishmal** [comment]
I don't have write access to merge PRs unless you mean to review the code.

**Comment by bgruening** [comment]
@kadishmal I guess @afshinm was asking you if you want to get commit access :)

**Comment by afshinm** [comment]
@kadishmal @bgruening Ah yes. If you want to be one maintainers, just let me know.

**Comment by afshinm** [comment]
@bgruening I have to say thanks to you. Without you this version of Intro.js was impossible. I will release a version today that addresses many issues.

**Comment by bgruening** [comment]
@afshinm no problem! I think it's a good idea to give more people commit access to share the burden. This is a great library and it deserves a great community!

**Comment by kadishmal** [comment]
@afshinm yes please. I can take a look at the MRs after you give me the write access.

**Comment by afshinm** [comment]
@kadishmal Please send me an email: afshin.meh@gmail.com

**Comment by kadishmal** [comment]
Sent.

**Comment by Fensterbank** [comment]
So, what's happening?
There are still so many issues and PRs and nobody cares.
